plugins/@grouparoo/pipedrive/__tests__/export/export-profile.ts

```diff
@@ -0,0 +1,410 @@
+  return filters[0];
+}
+
+async function cleanUp() {
+  // Clear created people
+  const ids = [];
+  for (let email of [email1, email2, email3, nonexistentEmail]) {
+    const id = await client.EnhancedPersonsController.findPersonIdByEmail(
+      email
+    );
+    if (id) {
+      ids.push(id);
+    }
+  }
+
+  if (ids.length > 0) {
+    await client.PersonsController.deleteMultiplePersonsInBulk(ids);
+  }
+
+  // Clear created group fields
+  const {
+    data: fields,
+  } = await client.PersonFieldsController.getAllPersonFields();
+  const testGroupFieldIds = fields
+    .filter((f) => [groupOneField, groupTwoField].includes(f.name))
+    .map((f) => f.id);
+
+  if (testGroupFieldIds.length > 0) {
+    await client.PersonFieldsController.deleteMultiplePersonFieldsInBulk(
+      testGroupFieldIds
+    );
+  }
+
+  // Clear created filters
+  const { data: filters } = await client.FiltersController.getAllFilters(
+    "people"
+  );
+  const testFilterIds = filters
+    .filter((f) => [groupOneField, groupTwoField].includes(f.name))
+    .map((f) => f.id);
+
+  if (testFilterIds.length > 0) {
+    await client.FiltersController.deleteMultipleFiltersInBulk(testFilterIds);
+  }
+}
+
+async function runExport({
+  oldProfileProperties,
+  newProfileProperties,
+  oldGroups,
+  newGroups,
+  toDelete,
+}) {
+  return exportProfile({
+    appOptions,
+    appId,
+    connection: null,
+    app: null,
+    destination: null,
+    destinationId: null,
+    destinationOptions: null,
+    export: {
+      profile: null,
+      profileId: null,
+      oldProfileProperties,
+      newProfileProperties,
+      oldGroups,
+      newGroups,
+      toDelete,
+    },
+  });
+}
+
+describe("pipedrive/exportProfile", () => {
+  beforeAll(async () => {
+    client = await connect(appOptions);
+    fieldMap = await getKnownPersonFieldMap(
+      client,
+      { appId, appOptions },
+      true
+    );
+    await cleanUp();
+  }, helper.setupTime);
+
+  afterAll(async () => {
+    await cleanUp();
+  }, helper.setupTime);
+
+  test("can create Person on pipedrive", async () => {
+    await runExport({
+      oldProfileProperties: {},
+      newProfileProperties: { Name: "John Doe", Email: email1 },
+      oldGroups: [],
+      newGroups: [],
+      toDelete: false,
+    });
+
+    const newPersonId = await client.EnhancedPersonsController.findPersonIdByEmail(
+      email1
+    );
+    expect(newPersonId).toBeTruthy();
+
+    personId = newPersonId;
+    const { data } = await client.PersonsController.getDetailsOfAPerson(
+      personId
+    );
+    expect(data.name).toBe("John Doe");
+    expect(data.email[0].value).toBe(email1);
+  });
+
+  test("can not create a Person without an email", async () => {
+    await expect(
+      runExport({
+        oldProfileProperties: {},
+        newProfileProperties: { Name: "Bill" },
+        oldGroups: [],
+        newGroups: [],
+        toDelete: false,
+      })
+    ).rejects.toThrow(/Email/);
+  });
```

Comment:
This technically could be any field, but we chose to keep it simple for now

From Pedro:
> You can search by any field https://developers.pipedrive.com/docs/api/v1/#!/Persons/searchPersons so technically we could use any field as an external id. Since we would be setting emails and Grouparoo (at the moment) doesn’t support multiple values for a field, we could also just use email.